processes/file/fulfill_url_manifest.py

```diff
@@ -16,15 +16,15 @@ class FulfillURLManifestProcess(CoreProcess):
     def __init__(self, *args):
         super(FulfillURLManifestProcess, self).__init__(*args[:4])
 
-        self.fullImport = self.process == 'complete' 
+        self.full_import = self.process == 'complete' 
         self.start_timestamp = None
 
         self.generateEngine()
         self.createSession()
 
-        self.s3Bucket = os.environ['FILE_BUCKET']
+        self.s3_bucket = os.environ['FILE_BUCKET']
         self.host = os.environ['DRB_API_HOST']
-        self.prefix = 'manifests/UofM/'
+        self.prefix = 'manifests/pubBacklist/'
```

Comment:
I recommend actually doing something like publisher_backlist to be explicit with the prefix. @Apophenia what are your thoughts?